Explorer/Assets/DCL/Multiplayer/Connections/Archipelago/SignFlow/LiveConnectionArchipelagoSignFlow.cs

```diff
@@ -112,7 +117,7 @@ public async UniTask SendHeartbeatAsync(Vector3 playerPosition, CancellationToke
 
                 await connection.SendAsync(clientPacket.value, memoryPool, token);
             }
-            catch (Exception e) { throw new Exception($"Cannot send heartbeat for position {playerPosition}", e); }
+            catch (Exception e) { ReportHub.LogException(new Exception($"Cannot send heartbeat for position {playerPosition}", e), ReportCategory.LIVEKIT); }
```

Comment:
I dont really like stopping the propagation of the exception here, in this class. I think it would be better to be handled on the consumer side. For example on this case, on the consumer, there is a particular exception handling for `ConnectionClosedException`, which will never happen with this changes

Reply:
True, it was a wrong exception management. Nonetheless we should improve it

Reply:
I've created an issue about this https://github.com/decentraland/unity-explorer/issues/1129 so we can tackle that later